contribs/gmf/apps/mobile_alt/Controller.js

```diff
@@ -58,4 +58,21 @@ const module = angular.module('Appmobile_alt', [appBase.name, gmfControllersAbst
 
 module.controller('AlternativeMobileController', Controller);
 
+module.constant('ngeoGeolocationOptions', {
+  positionFeatureStyle: {
+    circle: {
+      radius: 6,
+      fill: {color: 'rgba(230, 100, 100, 1)'},
+      stroke: {color: 'rgba(230, 40, 40, 1)', width: 2},
+    },
+  },
+  accuracyFeatureStyle: {
+    fill: {color: 'rgba(100, 100, 230, 0.3)'},
+    stroke: {color: 'rgba(40, 40, 230, 1)', width: 2},
+  },
+  zoom: 8,
+  autorotate: true,
+  atLoadingTime: true,
+});
```

Comment:
Steel should be removed...

Reply:
I don't understand where ``atLoadingTime: true`` then?

Reply:
Here: https://github.com/camptocamp/demo_geomapfish/blob/prod-2-6/geoportal/vars.yaml#L392